nodes/object/object_visibility_input.py

```diff
@@ -0,0 +1,32 @@
+import bpy
+from ... base_types.node import AnimationNode
+
+class an_ObjectVisibilityInput(bpy.types.Node, AnimationNode):
+    bl_idname = "an_ObjectVisibilityInput"
+    bl_label = "Object Visibility Input"
+    outputUseParameterName = "useOutput"
+    
+    def create(self):
+        self.inputs.new("an_ObjectSocket", "Object", "object").defaultDrawType = "PROPERTY_ONLY"
+        self.outputs.new("an_BooleanSocket", "Hide", "hide")
+        self.outputs.new("an_BooleanSocket", "Hide Select", "hideSelect").hide = True
+        self.outputs.new("an_BooleanSocket", "Hide Render", "hideRender")
+        self.outputs.new("an_BooleanSocket", "Show Name", "showName").hide = True
+        self.outputs.new("an_BooleanSocket", "Show Axis", "showAxis").hide = True
+        self.outputs.new("an_BooleanSocket", "Show Xray", "showXray").hide = True
+    
+    def getExecutionCode(self):
+
+        isLinked = self.getLinkedOutputsDict()
+        if not any(isLinked.values()): return ""
+        lines = []
+        lines.append("hide, hideSelect, hideRender, showName, showAxis, showXray = None, None, None, None, None, None")
```

Comment:
we have Booleans here. and a boolean shouldn't be None. Use False instead.
Maybe you can even write: hide = hideSelect = hideRender =...= False